### DISCUSSION TITLE
Is StoneDB a "real-time database"? Does it replace Firebase? thank you~

### DISCUSSION THREAD

**Discussion by 3xxx** [stoneatom/stonedb#635] [Q&A] [answered]
小白请教：请问这个算“实时数据库”吗？能替代firebase吗？谢谢。

**Comment by Nliver** [comment, marked as answer]
Yes,StoneDB is a real-time database.

As to whether StoneDB can replace FireBase, this needs to see the specific application scenario, specific case by case analysis~

**Reply by 3xxx** [reply]
thank you.
